dspace-swordv2/src/main/java/org/dspace/sword2/SwordUrlManager.java

```diff
@@ -515,4 +486,18 @@ public IRI getMediaFeedUrl(Item item)
         return new IRI(this.getSwordBaseUrl() + "/edit-media/" + item.getID() +
                            ".atom");
     }
+
+    /**
+     * Ensure configured paths when combined are vail URLs
+     *
+     * @param path the target SWORD endpoint
+     * @return a valid sword URL
+     */
+    private String validSWORDUrl(String path) {
```

Comment:
Same here, I think you can simplify this to simply be:
```
return dspaceUrl + "/" + swordPath + "/" + path;
```

Reply:
Simplified here as requested for the sword URL building: https://github.com/DSpace/DSpace/pull/3335/commits/032ffa6b41a6222c532c99107c5ef0327cdd33f3

Thanks for looking into it!